packages/contracts/deployments/README.md

```diff
@@ -8,18 +8,18 @@
 *The contracts relevant for the majority of developers are `OVM_ETH` and the cross-domain messengers. The L2 addresses don't change.*
 
 ### Predeploy contracts:
-|Contract|Address|
-|--|--|
-|OVM_ETH: | `0x4200000000000000000000000000000000000006`
-|OVM_L2StandardBridge: | `0x4200000000000000000000000000000000000010`
-|OVM_L2CrossDomainMessenger: | `0x4200000000000000000000000000000000000007`
-|OVM_L2ToL1MessagePasser: | `0x4200000000000000000000000000000000000000`
-|OVM_L1MessageSender: | `0x4200000000000000000000000000000000000001`
-|OVM_DeployerWhitelist: | `0x4200000000000000000000000000000000000002`
-|OVM_ECDSAContractAccount: | `0x4200000000000000000000000000000000000003`
-|OVM_SequencerEntrypoint: | `0x4200000000000000000000000000000000000005`
-|Lib_AddressManager: | `0x4200000000000000000000000000000000000008`
-|ERC1820Registry: | `0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24`
+| Contract                    | Address                                      |
```

Comment:
If this spacing was done by hand then it will be lost the next time that the script runs to autogen this file

Reply:
I actually have no idea how this spacing was "fixed", maybe the linter?